apps/journeys-admin/src/components/TemplateLibrary/TemplateLibrary.stories.tsx

```diff
@@ -13,18 +13,23 @@ const TemplateLibraryStory = {
   title: 'Journeys-Admin/TemplateLibrary'
 }
 
-const Template: Story = ({ ...args }) => (
-  <TemplateLibrary journeys={args.templates} />
-)
+const Template: Story = ({ ...args }) => <TemplateLibrary {...args} />
 
 export const Default = Template.bind({})
 Default.args = {
+  journeys: [{}],
```

Comment:
why are the curlies needed?
```suggestion
  journeys: [],
```

Reply:
Need to pass in something for journeys so we know they have access